src/firefly/js/util/fetch.js

```diff
@@ -6,13 +6,10 @@ import {getOrCreateWsConn} from '../core/messaging/WebSocketClient';
 import {ServerParams} from '../data/ServerParams';
 import {showInfoPopup} from '../ui/PopupUtil';
 import {DownloadProgress, getDownloadProgress} from '../rpc/SearchServicesJson';
+import {AJAX_REQUEST, doFetchUrl, REQUEST_WITH, WS_CHANNEL_HD, WS_CONNID_HD} from './lowerLevelFetch.js';
```

Comment:
It has to be in a new `lowerLevelFetch.js` file? Cannot just export `doFetchUrl` from within the same file?

Reply:
@loitly already talked about this, but for the record... `fetch.js` bring too much (react,logger,web socket stuff) to push into a worker. The worker grow to large. The lowLevelFetch.js only brings in fetch.  @loitly has suggested we change this file into some so of lowlevel util file. The might be worth doing, right now `WebUtil.js` mostly serves that purpose. Maybe it should be moved to `WebUtil.js`